client/my-sites/media-library/list-item.jsx

```diff
@@ -118,20 +120,22 @@ module.exports = React.createClass( {
 		return (
 			<div className={ classes } style={ style } onClick={ this.clickItem } { ...props }>
 				<span className="media-library__list-item-selected-icon">
-					<Gridicon icon="checkmark" size={ 20 } />
+					<Gridicon icon="checkmark" size={ 18 } />
```

Comment:
@aduth the linter for gridicons flagged this line for me - should I keep this change?